evm-bridging/scripts/Deploy.s.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.24;
+
+import {Script} from "forge-std/Script.sol";
+import {Upgrades} from "openzeppelin-foundry-upgrades/src/Upgrades.sol";
+import {BridgedTopShotMoments} from "../src/BridgedTopShotMoments.sol";
+
+contract DeployScript is Script {
+    function setUp() public {}
+
+    function run() external returns (address, address) {
+        // Start broadcast with deployer private key
+        vm.startBroadcast(vm.envUint("DEPLOYER_PRIVATE_KEY"));
+        console.log("Deployer address:", msg.sender);
+
+        // Set contract initialization parameters
+        address owner = msg.sender;
+        string memory name = "Bidged NBA TopShot Moments";
+        string memory symbol = "TOPSHOT";
+        string memory cadenceNFTAddress = "cadenceNFTAddress";
```

Comment:
do we get to change these values when deploying the bridge contract?

Reply:
Right, we should specify the proper values to be used when deploying. Also, only `symbol` currently has admin update method - we could add more if needed via contract update later, though I don't think we're expecting any of these to change